project/Build.scala

```diff
@@ -1341,12 +1341,14 @@ object Build {
       BuildInfoPlugin.buildInfoDefaultSettings
 
   def presentationCompilerSettings(implicit mode: Mode) = {
-    val mtagsVersion = "1.3.1"
+    val mtagsVersion = "1.3.1+63-1a8f4659-SNAPSHOT"
     Seq(
+      resolvers ++= Resolver.sonatypeOssRepos("snapshots"),
       libraryDependencies ++= Seq(
         "org.lz4" % "lz4-java" % "1.8.0",
         "io.get-coursier" % "interface" % "1.0.18",
         "org.scalameta" % "mtags-interfaces" % mtagsVersion,
+        "com.google.guava" % "guava" % "33.2.1-jre"
```

Comment:
@tgodzik Why is guava now necessary ? Shouldn't it be added transitevly from lsp ?